web/packages/url-utils/webpack/config.core.js

```diff
@@ -0,0 +1,66 @@
+const path = require( 'path' );
+const commonConfig = require( './config.common' );
+
+const baseConfig = {
+	...commonConfig,
+	entry: './src/index.ts',
+	output: {
+		path: path.resolve( 'dist' ),
+	},
+};
+
+const cjsConfig = {
+	...baseConfig,
+	output: {
+		...baseConfig.output,
+		filename: 'index.js',
+		library: {
+			type: 'commonjs2',
+		},
+	},
+	target: 'node',
+	optimization: {
+		...baseConfig.optimization,
+		minimize: false,
+	},
+};
+
+const esmConfig = {
+	...baseConfig,
+	output: {
+		...baseConfig.output,
+		filename: 'index.esm.js',
+		library: {
+			type: 'module',
+		},
+	},
+	experiments: {
+		outputModule: true,
+	},
+	optimization: {
+		...baseConfig.optimization,
+		minimize: false,
+	},
+};
+
+const mjsConfig = {
+	...esmConfig,
+	output: {
+		...esmConfig.output,
+		filename: 'index.mjs',
+	},
+};
+
+const umdConfig = {
+	...baseConfig,
+	output: {
+		...baseConfig.output,
+		filename: 'index.umd.js',
+		library: {
+			name: 'Gravatar',
+			type: 'assign-properties',
+		},
+	},
```

Comment:
Since we have supported `UMD`, maybe we could add [the relevant installation guide](https://github.com/Automattic/gravatar/blob/trunk/web/packages/hovercards/README.md#install-with-unpkg) in the `READMD.md`

Reply:
TIL about unpkg! Updated in 8b69588a6ec04c563a12c92026efb8b177a3d784